.circleci/config.yml

```diff
@@ -250,14 +250,43 @@ jobs:
     executor: docker-typescript-builder
     steps:
       - checkout
+      - aws-setup
+      - aws-ecr-setup
+      - local-deploy-setup
+      - run: echo "export IMAGE_TAG=dev_$(git rev-parse --short=8 HEAD)" >> $BASH_ENV
+      # since we're running with `--build-all`, assume that if it passes, we have all images required for Forge
+      - run: aws ecr describe-images --repository-name="aptos/validator" --image-ids=imageTag=$IMAGE_TAG
+      - run:
+          name: Start local testnet docker-compose
```

Comment:
docker-compose has to run on a machine type executor, e.g. `ubuntu-*` since it mounts a volume